pkg/kapp/diff/change_set_with_versioned_rs_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package diff
+
+import (
+	"testing"
+
+	ctlres "github.com/k14s/kapp/pkg/kapp/resources"
+)
+
+func TestChangeSet_ExistingVersioned_NewNonVersioned_Resource(t *testing.T) {
+	newRs, _ := ctlres.NewResourcesFromBytes([]byte(`
```

Comment:
use ctlres.MustNewResourceFromBytes so that errors do not get ignored. this method is useful in tests.